lib/src/models/metric_value_level.dart

```diff
@@ -0,0 +1,63 @@
+/// Enum class for value level.
+/// 
+/// Used when reporting computed metric value.
+class MetricValueLevel implements Comparable<MetricValueLevel> {
+  /// Value in "green" zone
+  ///
+  /// Default value
+  static const MetricValueLevel none = MetricValueLevel._('none');
```

Comment:
You can actually omit `MetricValueLevel` type from the declaration, like: 
`static const none = MetricValueLevel._('none');`